docs/documentation/docker.md

```diff
@@ -0,0 +1,182 @@
+# Docker
+
+## Navigating Docker Desktop/CLI
+
+- Install Docker on [Mac](https://docs.docker.com/desktop/setup/install/mac-install/), [Windows](https://docs.docker.com/desktop/setup/install/windows-install/), or [Linux](https://docs.docker.com/desktop/setup/install/linux/).
+
+Via Docker Desktop GUI,
+- Search for [sbryngelson/mfc](https://hub.docker.com/r/sbryngelson/mfc) repository where all the MFC images are stored then pull a release tag (e.g., `latest-cpu`).
+
+    Read through the **Tag Details** below to distinguish between them. Docker Desktop's left sidebar has two key tabs: **Images** stores your program copies, while **Containers** shows instances of those images. You can launch multiple containers from a single image.
+
+- Start a container by clicking the Run button in the Images tab.
+
+    Use the *Exec* section to interact with MFC directly via terminal, the *Files* section to transfer files between your device and container, and the *Stats* section to display resource usage.
+
+Or via Docker CLI,
+
+- Pull from [sbryngelson/mfc](https://hub.docker.com/r/sbryngelson/mfc) repository and run the latest MFC container.
+
+```bash
+docker run -it --rm --entrypoint bash sbryngelson/mfc:latest-cpu
+```
+<br>
+
+**Selecting OS/ARCH:** 
+
+Docker by default selects the compatible architecture when pulling and running a container. However, you can manually specify your platform (i.e., `linux/amd64` for most devices or `linux/arm64` for Apple Silicon).
+```bash
+docker run -it --rm --entrypoint bash --platform linux/amd64 sbryngelson/mfc:latest-cpu
```

Comment:
on Apple Silicon `docker run -it --rm --entrypoint bash sbryngelson/mfc:latest-cpu` works fine for me